daemon/api.go

```diff
@@ -2502,6 +2502,17 @@ func postDebug(c *Command, r *http.Request, user *auth.UserState) Response {
 		}, nil)
 	case "can-manage-refreshes":
 		return SyncResponse(devicestate.CanManageRefreshes(st), nil)
+	case "connectivity-check":
+		st.Unlock()
+		// FIXME: add to snapstate.StoreService? feels not ideal :/
+		//        other options?
+		s := getStore(c).(*store.Store)
```

Comment:
As you've already gotten the State from the command, and it's locked, don't use the helper here; use `snapstate.Store(st)` directly.